gpt_index/readers/download.py

```diff
@@ -14,9 +15,51 @@
 
 from gpt_index.readers.base import BaseReader
 
-LOADER_HUB_URL = (
-    "https://raw.githubusercontent.com/emptycrown/loader-hub/main/loader_hub"
-)
+LLAMA_HUB_CONTENTS_URL = "https://api.github.com/repos/ahmetkca/llama-hub/contents"
```

Comment:
We should change this back to the main repo

Reply:
Will do